tests/factors.spec.ts

```diff
@@ -153,7 +140,9 @@ export const FactorManipulationTest = async (testVariable: FactorTestVariable) =
       const instance = await newInstance();
       assert.strictEqual(instance.status, COREKIT_STATUS.LOGGED_IN);
 
-      instance.setTssWalletIndex(1);
+      if (instance.supportsAccountIndex) {
```

Comment:
hmm,
shouldn't the the accountSalt is generated during reconstruct tkey for existing user that do not have account index feature?

Reply:
not sure what you mean.

so what i'm doing here is:
1. check if instance supports account index.
2. only then try to set account index.

you think this should be done differently?